packages/rke2-coredns/generated-changes/overlay/templates/configmap-nodelocal.yaml

```
@@ -57,4 +57,14 @@ data:
         }
         prometheus :9253
         }
+    {{ range .Values.nodelocal.servers }}
+    {{- range $idx, $zone := .zones }}{{ $zone.zone }}{{ end -}}
+    {{- if .port }}:{{ .port }} {{ end -}}
+    {
+      {{- range .plugins }}
+        {{ .name }}
+          {{- end }}
```

nit - shouldn't this match the indentation of the `{{- range .plugins }}` line?
```suggestion
      {{- end }}
```

---

yes. fixed/